Library/ViewModels/PledgeViewModel.swift

```diff
@@ -276,14 +283,19 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       applePayStatusFailure
     )
 
-    // MARK: - Apple Pay
+    // MARK: - Create Apple Pay Backing
+
+    let willCreateApplePayBacking = Signal.combineLatest(
```

Comment:
This and `willUpdateApplePayBacking` are to ensure that upon completion of these asynchronous events we show the appropriate messaging/view to the user.